app/controllers/appointments_controller.rb

```diff
@@ -33,6 +44,13 @@ def show
 
   private
 
+  def date_range_params
+    starts = params[:start].to_date.beginning_of_day
+    ends   = params[:end].to_date.beginning_of_day
```

Comment:
It won't be missing unless the person hacks at the URL incorrectly. Also, that error doesn't get displayed in staging / production environments.